### DISCUSSION TITLE
How about a CONTRIBUTING file?

### DISCUSSION THREAD

**Discussion by BranAndSceolan** [MakeContributions/markdown-dungeon#215] [Ideas]
Hello everyone!

This project has many issues marked 'beginner' or 'good first issue'. I think that is great!
Maybe a CONTRIBUTING file with step-by-step explanations on how exactly people can contribute will give beginners extra confidence?
I imagine it containing explanations to questions like:
- I have found an issue I like. How do I start working on it?
- When should I use a fork?
- When should I use a branch?
- How do I make sure my pull request affects the correct branch?

These might be quite basic but beginners tend to feel overwhelmed by GitHubs many functions.

Or there could be a link to a guide explaining those things so that beginners know that this guide is okay to use.

**Comment by ming-tsai** [comment]
Hi @BranAndSceolan,
Here is the [contribution guideline](https://github.com/MakeContributions/markdown-dungeon#-contribution-guidelines) if you want to update it please let me know.
